libs/ast_generic/AST_generic.ml

```diff
@@ -1672,7 +1672,9 @@ and parameter =
   (* sgrep: ... in parameters
    * note: foo(...x) of Js/Go is using the ParamRest, not this *)
   | ParamEllipsis of tok
-  (* e.g., ParamTodo in OCaml, Reciever param in Go, SingleStar and Slash
+  (* Receiver param in Go *)
```

Comment:
maybe explain how the different from Param given they use the same type.
Is there an additional keyword with it? Does it use a special syntax?

Reply:
Done in https://github.com/returntocorp/semgrep/pull/7509